l10n_br_account_payment_brcobranca/models/account_payment_line.py

```diff
@@ -122,18 +122,21 @@ def _prepare_bank_line_santander(self, payment_mode_id, linhas_pagamentos):
     def prepare_bank_payment_line(self, bank_name_brcobranca):
         payment_mode_id = self.order_id.payment_mode_id
         linhas_pagamentos = self._prepare_boleto_line_vals()
-        try:
+
+        # Casos onde o Banco além dos principais campos possui campos
+        # específicos, dos casos por enquanto mapeados, se estiver vendo
+        # um caso que está faltando por favor considere fazer um
+        # PR para ajudar
+        if bank_name_brcobranca.name in (
+            "unicred",
+            "itau",
+            "banco_brasil",
+            "standader",
+        ):
```

Comment:
Se o objetivo é remover o bloco try-except não seria melhor apensar verificar se o método existe com hasattr?
Ai não teria necessidade de adicionar o nome do banco nessa lista, sempre que um método novo for criado.

Reply:
concordo com o @antoniospneto

Reply:
valeu @antoniospneto alterado para o hasattrs